src_c/constants.c

```diff
@@ -1056,6 +1056,22 @@ MODINIT_DEFINE(constants)
     DEC_CONST(APPACTIVE);
 #endif /* IS_SDLv2 */
 
+/* cursor constants */
+#if IS_SDLv2
+    DEC_CONST(SYSTEM_CURSOR_ARROW);
```

Comment:
Hi. Since you modified constants can you also modify [type hints](https://github.com/pygame/pygame/blob/master/buildconfig/pygame-stubs/constants.pyi) for it. You can just run the commented script at the top and you will get new file

Reply:
Got it. Thanks.